run/ktor-hello-world/.gitignore

```diff
@@ -0,0 +1,4 @@
+/.gradle/
+/.idea/
+/build/
+*.iml
```

Comment:
We already [have a `.gitignore`](https://github.com/GoogleCloudPlatform/kotlin-samples/blob/master/.gitignore) which covers the same items here. Kinda up to you if we need this or not, since if someone copies this directory over, including the `.gitignore` is useful. But if copying this directory over is not the intended purpose, then I'd advocate removing this.

Reply:
I've tried to make it so they can copy the sub-dir so I think we should keep this.